hack/e2e-suite/guestbook.sh

```diff
@@ -38,30 +38,78 @@ GUESTBOOK="${KUBE_ROOT}/examples/guestbook"
 function teardown() {
   ${KUBECTL} stop -f "${GUESTBOOK}"
   if [[ "${KUBERNETES_PROVIDER}" == "gce" ]]; then
-    gcloud compute forwarding-rules delete "${INSTANCE_PREFIX}-default-frontend" || true
-    gcloud compute target-pools delete "${INSTANCE_PREFIX}-default-frontend" || true
+    gcloud compute forwarding-rules delete "${INSTANCE_PREFIX}-default-frontend" --region="${ZONE}" -q || true
+    gcloud compute target-pools delete "${INSTANCE_PREFIX}-default-frontend" --region="${ZONE}" -q || true
+    gcloud compute firewall-rules delete guestbook-e2e-minion-8000 -q || true
   fi  
 }
 
+function wait_for_running() {
+  echo "Waiting for pods to come up."
+  frontends=$(${KUBECTL} get pods -l name=frontend -o template '--template={{range.items}}{{.id}} {{end}}')
```

Comment:
Declare these variables as local and do it in its own line (no assignment to `$(...)` on the same line as `local` as that hides the error exit of the command.)

```shell
  local frontends master slaves pods all_running status i pod
  frontends=$(...)
  ...
```